index.ts

```diff
@@ -1,3 +1,4 @@
+export * as isoly from "./isoly"
 import { CallingCode } from "./CallingCode"
```

Comment:
I guess the old ones are preservred for backward compability.
Put a comment about the double ways to import, and which one that is prefered.